.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -85,6 +85,7 @@ body:
         **Note**: for ECMASCript modules (ESM) in Node, at minimum Node.js 12.20, 14.14, or 16.0 is required.
       multiple: true
       options:
+        - Node v20
         - Node v19
         - Node v18
         - Node v17
```

Comment:
I think we might as well remove Node v12, Node v14, Node v17, and Node V19, and add Node v21 while we’re at it.

Reply:
> All legacy versions are just unsupported 

Yes, legacy versions are unsupported.

> right after new version released?

With each major release we remove Node versions the Node foundation does not support.

> Is this too strict?

No, legacy support is a massive time sink.
Speaking for myself, I have neither the time nor interest in supporting legacy Node versions.

> For example, we've published a v1 hotfix for eslint-mdx at https://github.com/mdx-js/eslint-mdx/tree/release-v1 after v2 has been released days after.

Okay? How does this relate to the versions of Node we list in the bug template?

Reply:
We should list old ones. It makes it easier to see if people are using old versions.

Same reason planes have ashtrays. Some people are going to do it anyway.